### ISSUE TITLE
Remote language server is not working until identity is checked

### ISSUE THREAD

**Issue by nvierge** [kaby76/AntlrVSIX#39]
I encounter a weird problem: the activation of the remote language server was not working. As a consequence i did not benefit from syntax highlighing. After some debugging, i notice an exception about identity that cannot be checked with the domain. So i activate my VPN and now it works...
Why do you need to check user identity to start the language server ?
The lack of logs make the diagnostic quite hard.

**Comment by kaby76** [comment]
I assume that is for the code I added to blacklist certain id's. I've had second thoughts about this. I thought I made it dead code but it makes the call to get the identity, which crashes before starting the server. I will remove the check because I was rash. Please understand that I have been hurt quite a bit having been unemployed for 20 years, having applied and rejected without even the courtesy of rejection letter. This is a terrible profession.

There is a logging system, but it specifically goes to a server of mine, not to a file on your file system. I tried using Nlog, but I couldn't get it to work. I can output tracebacks to a file always, if you would find that useful.

**Comment by nvierge** [comment]
I am not an expert in vs extension development but the activity log could be an option to consider to log errors.

**Comment by kaby76** [comment]
I'm no expert either. But, errors used to be logged to a VS log file. But that was very annoying because a lot of errors weren't fatal, just unhandled situations in the logic of the code. Rather than burden folks with this errors, I added code to catch and send the error onto a server (http://domemtech.com/home/db) for me to look at. But, I see that was probably wasn't a great idea. (I haven't even looked that the db for a while.) I've added some code to log to ~/.antlrlog for the VS session for the time being until I figure out a better solution.
